docs/Technical-Documentation/TDP-environments-README.md

```diff
@@ -0,0 +1,73 @@
+# TDP Environments
+
+## Development
+
+| Dev Site | Frontend URL | Backend URL | Purpose |
+| -------- | -------- | -------- | -------- |
+| Sandbox     | https://tdp-frontend-sandbox.app.cloud.gov | https://tdp-backend-sandbox.app.cloud.gov     | Space for devs to test in a deployed environment 
+| A11y | https://tdp-frontend-a11y.app.cloud.gov | https://tdp-backend-a11y.app.cloud.gov | Space for accessibility testing |
+| QASP | https://tdp-frontend-qasp.app.cloud.gov | https://tdp-backend-qasp.app.cloud.gov | Space for QASP review |
+| raft | https://tdp-frontend-raft.app.cloud.gov | https://tdp-backend-raft.app.cloud.gov | Space for raft review |
+
+### Dependencies 
+
+- `clamav-rest`
+
+**Cloud.gov AWS RDS `(tanf-dev)`**
+- `tdp-db-dev`
+  
+**Cloud.gov AWS S3 `(tanf-dev)`**
+- `tdp-staticfiles-dev`
+- `tdp-datafiles-dev`
```

Comment:
any sample raw data files to be stored here?

Reply:
I will clarify in the README – `datafiles` will store uploaded files from STTs. Staticfiles will stre Django admin HTML/CSS.